lib/eqaf_bytes.ml

```diff
@@ -0,0 +1,32 @@
+let equal a b =
+  let a' = Bytes.unsafe_to_string a in
+  let b' = Bytes.unsafe_to_string b in
+  Eqaf.equal a' b'
+
+let compare_le_with_len ~len a b =
+  let a' = Bytes.unsafe_to_string a in
+  let b' = Bytes.unsafe_to_string b in
+  Eqaf.compare_le_with_len ~len a' b'
+
+let compare_le a b =
+  let a' = Bytes.unsafe_to_string a in
+  let b' = Bytes.unsafe_to_string b in
+  Eqaf.compare_le a' b'
+
+let compare_be_with_len ~len a b =
+  let a' = Bytes.unsafe_to_string a in
+  let b' = Bytes.unsafe_to_string b in
+  Eqaf.compare_be_with_len ~len a' b'
+
+let compare_be a b =
+  let a' = Bytes.unsafe_to_string a in
+  let b' = Bytes.unsafe_to_string b in
+  Eqaf.compare_be a' b'
+
+let find_uint8 ?(off= 0) ~f b =
```

Comment:
is `= 0` necessary here?

Reply:
You can indeed keep the optional argument and give it as is to `Eqaf.find_uint8` instead of specifying `0`